features/unit/responses.feature

```diff
@@ -63,6 +63,84 @@ Feature: REST Client Responses
+      | indexer_unlimited_aa_TransactionResponse_17.json           | generated_responses_unlimited_assets | 200    | indexer | lookupTransaction                |
+      | indexer_unlimited_aa_TransactionsResponse_0.json           | generated_responses_unlimited_assets | 200    | indexer | lookupAccountTransactions        |
+      | indexer_unlimited_aa_TransactionsResponse_0.json           | generated_responses_unlimited_assets | 200    | indexer | lookupAssetTransactions          |
+      | indexer_unlimited_aa_TransactionsResponse_0.json           | generated_responses_unlimited_assets | 200    | indexer | searchForTransactions            |
+      | indexer_unlimited_aa_TransactionsResponse_1.json           | generated_responses_unlimited_assets | 200    | indexer | lookupAccountTransactions        |
+      | indexer_unlimited_aa_TransactionsResponse_1.json           | generated_responses_unlimited_assets | 200    | indexer | lookupAssetTransactions          |
+      | indexer_unlimited_aa_TransactionsResponse_1.json           | generated_responses_unlimited_assets | 200    | indexer | searchForTransactions            |
+      | indexer_unlimited_aa_TransactionsResponse_2.json           | generated_responses_unlimited_assets | 200    | indexer | lookupAccountTransactions        |
+      | indexer_unlimited_aa_TransactionsResponse_2.json           | generated_responses_unlimited_assets | 200    | indexer | lookupAssetTransactions          |
+      | indexer_unlimited_aa_TransactionsResponse_2.json           | generated_responses_unlimited_assets | 200    | indexer | searchForTransactions            |
+      | indexer_unlimited_aa_ErrorResponse_0.json                  | generated_responses_unlimited_assets | 500    | indexer | any                              |
+      | indexer_unlimited_aa_ErrorResponse_1.json                  | generated_responses_unlimited_assets | 500    | indexer | any                              |
+      | algod_unlimited_aa_Account_0.json                          | generated_responses_unlimited_assets | 200    | algod   | AccountInformation               |
+      | algod_unlimited_aa_Account_2.json                          | generated_responses_unlimited_assets | 200    | algod   | AccountInformation               |
+      | algod_unlimited_aa_AccountApplicationResponse_0.json       | generated_responses_unlimited_assets | 200    | algod   | AccountApplicationInformation    |
+      | algod_unlimited_aa_AccountApplicationResponse_2.json       | generated_responses_unlimited_assets | 200    | algod   | AccountApplicationInformation    |
+      | algod_unlimited_aa_AccountAssetResponse_0.json             | generated_responses_unlimited_assets | 200    | algod   | AccountAssetInformation          |
+      | algod_unlimited_aa_AccountAssetResponse_2.json             | generated_responses_unlimited_assets | 200    | algod   | AccountAssetInformation          |
```

Comment:
@algonautshant do you know why we have 0 and 2 for these?

Reply:
Because of https://github.com/algorand/generator/pull/25. Please approve and merge.

Reply:
nope. #25 did not fix it. Looking into it.

Reply:
I read toliks message as #25 explains the behavior.

Reply:
I have a fix. It is in the logic of accounting for num. It just looks strange, but can be fix later, since this is internal test.